### DISCUSSION TITLE
SSL support for UI

### DISCUSSION THREAD

**Discussion by cugwmui** [alephdata/aleph#2844] [Q&A] [answered]
I'm wondering if there is any way to have SSL enabled access? The current dockerfile only has http for nginx whereas integration with OAUTH/identify providers like Amazon Cognito mandates SSL based callback URLs.
Any suggestions on how to proceed with this?

**Comment by tillprochaska** [comment]
Common solutions are a reverse proxy in front of Aleph or using a managed service that handles TLS termination for you. This depends on your hosting/deployment setup, so we’re unfortunately not able to provide more specific instructions.

**Comment by cugwmui** [comment, marked as answer]
Thanks. I ultimately configured it with a SSL termination proxy.
Ideally a solution would be a minor set of changes in the nginx.conf and definition of an SSL certificate, but solutions like reverse proxies work too.
